examples/siro_sandbox/README.md

```diff
@@ -0,0 +1,136 @@
+# Sandbox Tool
+
+![siro_sandbox_screenshot](https://user-images.githubusercontent.com/6557808/230213487-f4812c2f-ec7f-4d68-9bbe-0b65687f769b.png)
+
+# Overview
+This is a 3D interactive GUI app for testing various pieces of SIRo, e.g. rearrangement episode datasets, Fetch and Spot robots, humanoids (controllers, animation, skinning), trained agent policies, batch rendering and other visualization.
```

Comment:
Should this description be SIRo-specific?

Reply:
Habitat 3.0?